model/metrics/nodejs-metrics.yaml

```diff
@@ -73,9 +73,23 @@ groups:
     type: metric
     metric_name: nodejs.eventloop.utilization
     brief: "Event loop utilization."
+    instrument: gauge
+    unit: "1"
+    stability: experimental
+    note: >
+      The value range is [0.0,1.0] and can be retrieved from `utilization` from
+      [`performance.eventLoopUtilization([utilization1[, utilization2]])`](https://nodejs.org/api/perf_hooks.html#performanceeventlooputilizationutilization1-utilization2)
+
+  - id: metric.nodejs.eventloop.time
+    type: metric
+    metric_name: nodejs.eventloop.time
+    brief: "Cumulative duration of time the event loop has been on the state."
     instrument: counter
     unit: "s"
     stability: experimental
+    attributes:
+      - ref: nodejs.eventloop.type
```

Comment:
I don't feeling strong about it, but please consider `nodejs.eventloop.state` which seems to work better with `active` and `idle` values. I'd expect something else under type (like specific implementation name).

Reply:
+1 to the naming change. Looking at some refs within semconv I see the concept of `mode` for [cpu time](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/system-metrics.md#metric-systemcputime) or `state` for [memory usage](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/system-metrics.md#metric-systemcputime)

Reply:
Updated